src/components/submission/analyze.vue

```diff
@@ -45,33 +53,22 @@ except according to the terms contained in the LICENSE file.
       <div id="odata-analyze-tool-help" class="modal-introduction">
         <i18n-t v-if="tool === 'microsoft'" tag="p" keypath="help.microsoft.full">
           <template #pageForExcel>
-            <a href="https://support.office.com/en-us/article/connect-to-an-odata-feed-power-query-4441a94d-9392-488a-a6a9-739b6d2ad500" target="_blank" rel="noopener">{{ $t('help.microsoft.pageForExcel') }}</a>
+            <a href="https://support.microsoft.com/en-us/office/about-power-query-in-excel-7104fbee-9e62-4cb9-a02e-5bfb1a6c536a" target="_blank" rel="noopener">{{ $t('help.microsoft.pageForExcel') }}</a>
           </template>
           <template #pageForPowerBi>
-            <a href="https://docs.microsoft.com/en-us/power-bi/desktop-connect-odata" target="_blank" rel="noopener">{{ $t('help.microsoft.pageForPowerBi') }}</a>
+            <a href="https://learn.microsoft.com/en-us/power-bi/connect-data/desktop-connect-odata" target="_blank" rel="noopener">{{ $t('help.microsoft.pageForPowerBi') }}</a>
+          </template>
+        </i18n-t>
+        <i18n-t v-else-if="tool === 'python'" tag="p" keypath="help.python">
+          <template #pyODK>
+            <a href="https://github.com/getodk/pyodk/" target="_blank" rel="noopener">PyODK</a>
```

Comment:
PyODK → pyODK? How do we stylize the name?

Reply:
Ah yea, it should just be pyODK.